### PR TITLE
Added the new permissions in the permissions reference doc

### PR DESCRIPTION
**Pull request by Y-Meenakshi**
Hi @nkramer, @laujan,
Please find the updated permission reference doc with my part of additions. Kindly review.

### CODE REVIEW

File: concepts/permissions-reference.md [hunk 1]
```diff
@@ -90,7 +90,7 @@ The following is the JSON response:
 
 ---
 
-## Access reviews permissions
```
Comment (nkramer) [inline concepts/permissions-reference.md:93]: revert

File: concepts/permissions-reference.md [hunk 2]
```diff
@@ -253,12 +254,98 @@ The _Application.ReadWrite.OwnedBy_ permission allows the same operations as _Ap
 
 ---
 
-## Bookings permissions
+## AppRole assignment permissions ([private preview](#permissions-availability-status))
```
Comment (nkramer) [inline concepts/permissions-reference.md:257]: not us

File: concepts/permissions-reference.md [hunk 3]
```diff
@@ -253,12 +254,98 @@ The _Application.ReadWrite.OwnedBy_ permission allows the same operations as _Ap
...
+| _ApprovalRequest.ReadWrite.AdminConsentRequest_ (private preview) | Read and write admin consent approval requests | Allows the app to read and write admin consent requests, business flows, and governance policy templates on behalf of the signed-in user. | Yes | No |
+| _ApprovalRequest.ReadWrite.CustomerLockbox_ (private preview) | Read and write customer lockbox approval requests | Allows the app to read and write customer lockbox requests, business flows and governance policy templates on behalf of the signed-in user. | Yes | No |
+| _ApprovalRequest.ReadWrite.EntitlementManagement_ (private preview) | Read and write entitlement management approval requests | Allows the app to read and write entitlement management requests, business flows, and governance policy templates on behalf of the signed-in user. Yes | No |
+| _ApprovalRequest.ReadWrite.PriviligedAccess_ (private preview) | Read and write privileged access approval request | Allows the app to read and write privileged access requests, business flows, and governance policy templates on behalf of the signed-in user. | Yes | No |
+
+#### Application permissions
```
Comment (nkramer) [inline concepts/permissions-reference.md:294]: not us

File: concepts/permissions-reference.md [hunk 4]
```diff
@@ -253,12 +254,98 @@ The _Application.ReadWrite.OwnedBy_ permission allows the same operations as _Ap
...
+| _ApprovalRequest.ReadWrite.AdminConsentRequest_ (private preview) | Read and write all admin consent approval requests | Allows the app to read and write admin consent requests, business flows, and governance policy templates without a signed-in user. | Yes |
+| _ApprovalRequest.ReadWrite.CustomerLockbox_ (private preview) | Read and write all customer lockbox approval requests | Allows the app to read and write customer lockbox requests, business flows, and governance policy templates without a signed-in user.| Yes |
+| _ApprovalRequest.ReadWrite.EntitlementManagement_ (private preview) | Read and write all entitlement management approval requests | Allows the app to read and write entitlement management requests, business flows, and governance policy templates without a signed-in user. | Yes |
+| _ApprovalRequest.ReadWrite.PriviligedAccess_ (private preview) | Read and write all privileged access approval requests | Allows the app to read and write privileged access requests, business flows, and governance policy templates without a signed-in user. | Yes |
+
+---
+
+## Audit log permissions ([private preview](#permissions-availability-status))
```
Comment (nkramer) [inline concepts/permissions-reference.md:309]: not us

File: concepts/permissions-reference.md [hunk 5]
```diff
@@ -253,12 +254,98 @@ The _Application.ReadWrite.OwnedBy_ permission allows the same operations as _Ap
...
+
+#### Application permissions
+
+|   Permission    |  Display String   |  Description | Admin Consent Required | Microsoft Account supported |
+|:----------------|:------------------|:-------------|:-----------------------|:--------------|
+| _AuditLog.Read.All_ (private preview) | Read all audit log data | Allows the app to read and query your audit log activities, without a signed-in user. | Yes |
+
+---
+
+## Bitlocker key permissions ([private preview](#permissions-availability-status))
```
Comment (nkramer) [inline concepts/permissions-reference.md:325]: not us

File: concepts/permissions-reference.md [hunk 6]
```diff
@@ -253,12 +254,98 @@ The _Application.ReadWrite.OwnedBy_ permission allows the same operations as _Ap
...
+#### Application permissions
+
 |   Permission    |  Display String   |  Description | Admin Consent Required | Microsoft Account supported |
 |:----------------|:------------------|:-------------|:-----------------------|:--------------|
+| _BitlockerKey.Read.All_ (private preview) | Read all BitLocker keys | Allows an app to read BitLocker keys for all devices, without a signed-in user. Allows read of the recovery key. | Yes | No |
+| _BitlockerKey.ReadBasic.All_ (private preview) | Read all BitLocker keys basic information | Allows an app to read basic BitLocker key properties for all devices, without a signed-in user. Does not allow read of the recovery key. | Yes | No |
+
+---
+
+## Bookings permissions
```
Comment (nkramer) [inline concepts/permissions-reference.md:343]: not us

File: concepts/permissions-reference.md [hunk 7]
```diff
@@ -454,28 +547,43 @@ For more complex scenarios involving multiple permissions, see [Permission scena
...
-|_Chat.Send.All_ (private preview)|Send user chat messages  |Allows an app to send 1:1 and group chat messages in Microsoft Teams without a signed-in user. |No | No |
+| _Chat.ReadWrite.All_ (private preview) | Read and write all chat messages | Allows an app to read and write all one-to-one and group chats in Microsoft Teams, without a signed-in user. Does not allow sending messages. | Yes | No |
+| _Chat.UpdatePolicyViolation.All_ |Flag chat messages for violating policy |Allows the app to update Microsoft Teams 1:1 or group chat messages by patching a set of Data Loss Prevention (DLP) policy violation properties to handle the output of DLP processing. | Yes | No |
+| _Chat.Send.All_ (private preview)|Send user chat messages  |Allows an app to send 1:1 and group chat messages in Microsoft Teams without a signed-in user. |No | No |
 
 > **Note:** For messages in a channel, see [ChannelMessage permissions](#channel-message-permissions).
 
+---
+
+## Consents request permissions ([private preview](#permissions-availability-status))
```
Comment (nkramer) [inline concepts/permissions-reference.md:576]: not us

File: concepts/permissions-reference.md [hunk 8]
```diff
@@ -601,14 +720,32 @@ For more complex scenarios involving multiple permissions, see [Permission scena
 
 ---
 
-## Domain permissions
```
Comment (nkramer) [inline concepts/permissions-reference.md:604]: not us

File: concepts/permissions-reference.md [hunk 9]
```diff
@@ -601,14 +720,32 @@ For more complex scenarios involving multiple permissions, see [Permission scena
...
+| _Domain.ReadWrite.All_ (private preview) | Read and write domains | Allows the app to read and write all domain properties on behalf of the signed-in user. Also allows the app to add, verify and remove domains. | Yes | No |
+
 
 #### Application permissions
 
 |   Permission    |  Display String   |  Description | Admin Consent Required |
 |:-----------------------------|:-----------------------------------------|:-----------------|:-----------------|
 | _Domain.ReadWrite.All_ | Read and write domains | Allows the app to read and write domains without a signed-in user. | Yes |
 
+---
```
Comment (nkramer) [inline concepts/permissions-reference.md:739]: not us

File: concepts/permissions-reference.md [hunk 10]
```diff
@@ -652,14 +789,32 @@ For more complex scenarios involving multiple permissions, see [Permission scena
 
 ---
 
-## Entitlement management permissions
```
Comment (nkramer) [inline concepts/permissions-reference.md:655]: not us

File: concepts/permissions-reference.md [hunk 11]
```diff
@@ -652,14 +789,32 @@ For more complex scenarios involving multiple permissions, see [Permission scena
...
 |Permission|Display String|Description|Admin Consent Required|
 |:----------|:--------------|:-----------|:-------|
-|_EntitlementManagement.ReadWrite.All_|Read and write entitlement management resources|Allows the app to request access to read and manage access packages and related entitlement management resources on behalf of the signed-in user.|Yes|
-|_EntitlementManagement.Read.All_|Read entitlement management resources|Allows the app to request access to read access packages and related entitlement management resources on behalf of the signed-in user.|Yes|
+|_EntitlementManagement.ReadWrite.All_ (private preview)|Read and write entitlement management resources|Allows the app to request access to read and manage access packages and related entitlement management resources on behalf of the signed-in user.|Yes|
+|_EntitlementManagement.Read.All_ (private preview)|Read entitlement management resources|Allows the app to request access to read access packages and related entitlement management resources on behalf of the signed-in user.|Yes|
+
+---
+
+## EWS access as user permissions ([private preview](#permissions-availability-status))
```
Comment (nkramer) [inline concepts/permissions-reference.md:803]: not us

File: concepts/permissions-reference.md [hunk 12]
```diff
@@ -652,14 +789,32 @@ For more complex scenarios involving multiple permissions, see [Permission scena
...
+
+#### Delegated permissions
+|   Permission    |  Display String   |  Description | Admin Consent Required | Microsoft Account supported |
+| _EWS.AccessAsUser.All_ (private preview) | Access mailboxes as the signed-in user via Exchange Web Services | Allows the app to have the same access to mailboxes as the signed-in user via Exchange Web Services. | No | No|
+
+---
+
+## Family permissions ([private preview](#permissions-availability-status))
```
Comment (nkramer) [inline concepts/permissions-reference.md:811]: not us

File: concepts/permissions-reference.md [hunk 13]
```diff
@@ -771,7 +926,7 @@ For more complex scenarios involving multiple permissions, see [Permission scena
 ---
 
 
-## Identity provider permissions
```
Comment (nkramer) [inline concepts/permissions-reference.md:774]: not us

File: concepts/permissions-reference.md [hunk 14]
```diff
@@ -799,13 +961,14 @@ For more complex scenarios involving multiple permissions, see [Permission scena
 
 ---
 
-## Identity risk event permissions
+## Identity risk event permissions ([private preview](#permissions-availability-status))
```
Comment (nkramer) [inline concepts/permissions-reference.md:964]: not us

File: concepts/permissions-reference.md [hunk 15]
```diff
@@ -868,6 +1030,46 @@ For more complex scenarios involving multiple permissions, see [Permission scena
 
 ---
 
+## Identity user flow permissions ([private preview](#permissions-availability-status))
```
Comment (nkramer) [inline concepts/permissions-reference.md:1033]: not us

**Comment by Y-Meenakshi** [comment]
Merged

**Pull request closed without merging**